### DISCUSSION TITLE
感謝貼

### DISCUSSION THREAD

**Discussion by icm-ai** [laubonghaudoi/jyutping.org#59] [General]
最近學粵拼，從[粵拼JYUTPING](https://jyutping.org/docs/cantonese/)過來的，這個網站給我的感覺就像是一個github項目，拉到最下面果然是，star了，非常感謝團隊的工作，我也是一個coder，目前專注於AI，希望有機會合作。

**Comment by laubonghaudoi** [comment]
感謝你嘅支持，歡迎同其他人分享我哋嘅作品。我哋嘅官方Twitter係 https://x.com/Can_CLID ，歡迎關注獲取我哋作品嘅最新消息。
